source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -222,22 +274,67 @@ def _read_response(response, origin, destination, mode, fallback_extremity, requ
 
         return resp
 
-    def get_direct_path_params(self, origin, destination, mode, fallback_extremity):
+    def get_realtime_traffic_parameter(self, request):
```

Comment:
This logic is exactly the same as the one in the Here class' constructor. Instead of repeating yourself, you can re-use this function up there by refactoring it a little :

```suggestion
    def get_realtime_traffic_parameter(self, realtime_traffic):
        [...]
```